configserv/image.yaml

```diff
@@ -1,4 +1,4 @@
-version: 1.0
+version: latest
```

Comment:
I'm happy to explain it.

The Red Hat official build system adds additional tags. One of them is `latest`. This should be the case for any build system that you use. This applies to our CI too. I suggest here using proper versions in testing too - we noticed that it's very easy to mix images and sometimes QE was testing an image version that shouldn't be tested. Using "latest" tag will make it even harder.

When you define the main version as "latest" should be considered wrong. This should be your target version, which is currently `1.0`. As long as you develop your images - the version won't change and will be always `1.0`. In other words - your images should be always prepared to be released - since if you change the source code, you would need to build it again and it would burn QE time again, and so on.

Tags like "latest" are something that is added to an image, after the image is actually built. With tags you control what it what, but the main version needs to be defined properly.

Hope this is now more clear.